src/app/components/connect-addon-modal/connect-addon-modal.html

```diff
@@ -0,0 +1,82 @@
+<div id="connect-addon-modal">
+  <div class="modal-header">
+    <div class="close" type="button" aria-label="Close" ng-click="$ctrl.close()">×</div>
+    <h3 class="modal-title"
+        translate="mno_enterprise.webhook.o_auth.providers.xero.connect_to"
+        translate-values="{appname: $ctrl.app.name}"></h3>
+  </div>
+
+  <!-- Link -->
+  <div ng-show="!$ctrl.hasLinked && !$ctrl.hasChosenEntities">
+    <ul class="breadcrumb">
+      <li>{{"mno_enterprise.templates.components.addon_connect.link_account.title" | translate}}</li>
+      <li><a href="#">{{"mno_enterprise.templates.components.addon_connect.entities.select" | translate}}</a></li>
+      <li><a href="#">{{"mno_enterprise.templates.components.addon_connect.sync.start_sync" | translate}}</a></li>
```

Comment:
This CSS class deactivate the ability to click a link:
```
.not-active {
   pointer-events: none;
   cursor: default;
}
```

Reply:
Also, this code should be moved outside the `ng-show` and use a variable to store the active step.